ui/app/mixins/unload-model-route.js

```diff
@@ -19,7 +19,6 @@ export default Mixin.create({
       return;
     }
     removeRecord(this.store, model);
-    model.destroy();
```

Comment:
This doesn't seem to be needed and removing it doesn't seem to change anything other than get rid of some errors in tests. I believe that calling `unloadRecord` is sufficient and ember data should take care of the rest.